pkg/platform/docker/util.go

```diff
@@ -157,14 +206,18 @@ func getAlias(ctx context.Context, logger *zap.Logger, opts models.SetupOptions)
 			alias += " --proxy-port " + fmt.Sprintf("%d", opts.ProxyPort)
 			return alias, nil
 		}
-		// if default docker context is used
-		logger.Info("Starting keploy in docker with default context, as that is the current context.")
+		// Handle default and other contexts
+		if activeContext == "default" {
```

Comment:
There is no need for two different logs, You can just log that keploy is starting with this context ( improve the messaging a bit ).